front/pages/solutions/customer-support.tsx

```diff
@@ -53,81 +53,72 @@ export default function CustomerSupport() {
   return (
     <>
       <HeaderContentBlock
-        uptitle="Dust for Customer Support"
-        title={
-          <>
-            Help your Support&nbsp;Team,
-            <br />
-            help your&nbsp;customers.
-          </>
-        }
+        uptitle="Dust for Success and&nbsp;Support&nbsp;Teams"
+        title="Help your Success and&nbsp;Support&nbsp;Teams, help&nbsp;your&nbsp;customers."
         from="from-sky-200"
         to="to-sky-500"
-        subtitle={
-          <>
-            Reply faster and increase answer&nbsp;quality
-            <br />
-            by augmenting your&nbsp;Customer&nbsp;Support with
-            AI&nbsp;assistants.
-          </>
-        }
+        subtitle="Equip your&nbsp;team with AI&nbsp;assistants to&nbsp;accelerate
+            issue resolution and&nbsp;increase customer&nbsp;satisfaction."
       />
       <Grid>
         <SolutionSection
           title={
             <>
-              Happy Agents,
+              Elevate Support,
               <br />
-              happy Customers
+              exceed Customer&nbsp;expectations.
             </>
           }
           blocks={[
             {
               color: "sky",
               contentBlocks: [
                 {
-                  title: (
+                  title: "Keep your team spot-on.",
```

Comment:
Let's try to be consistent and either we use a string, or we use a `ReactNode` but let's not mix both.

Reply:
Here for instance:

<img width="558" alt="image" src="https://github.com/dust-tt/dust/assets/4435185/fd2907d2-5717-4d3a-92d0-3d3820818558">